config.TEMPLATE.inc.php

```diff
@@ -34,6 +34,10 @@
 ; Session cookie name
 session_cookie_name = OJSSID
 
+; Session cookie domain, for example "pkp.sfu.ca"
+; If not specified, defaults to the detected server host
+; session_cookie_domain = 
```

Comment:
The comment should include a description when and why someone may wish to use this config option. Feel free to get detailed.